### PR TITLE
Prepare a 0.4.0 release.

### PR DESCRIPTION
**Pull request by ltratt**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -23,6 +23,18 @@
   `snare` informs users whose config contains `email` how to update to
   `errorcmd` to obtain the previous behaviour.
 
+## Minor changes
+
+* After daemonisation, all errors are now sent to syslog (previously a few
+  errors could still be sent to stderr).
+
+* Fix bug in parsing string escapes, where one character too many was
+  consumed after `\"`.
```
Comment (vext01) [inline CHANGES.md:32]: Just to check, this is the intended rendering for that verbatim part:
image

Correct?

**Comment by vext01** [comment]
One small question, otherwise good.

File: CHANGES.md (hunk 1 above)
Comment (ltratt) [reply CHANGES.md:32]: Yep!

**Pull request merged**